spec/consensus/proposer-based-timestamp/tla/TendermintPBT_002_draft.tla

```diff
@@ -810,6 +777,13 @@ AgreementOnValue ==
             /\ decision[p] = Decision(prop, r1)
             /\ decision[q] = Decision(prop, r2)
 
+DisagreementOnValue ==
+  \E p, q \in Corr:
+    \E p1 \in ValidProposals, p2 \in ValidProposals, r1 \in Rounds, r2 \in Rounds:
+    /\ p1 /= p2
+    /\ decision[p] = Decision(p1, r1)
+    /\ decision[q] = Decision(p2, r2) 
+
 \* [PBTS-CONSENSUS-TIME-VALID.0]
```

Comment:
Assuming you're talking about `[PBTS-CONSENSUS-TIME-VALID.0]` and not the lines from `DisagreementOnValue`, you might be right, but it's not something that has changed in this PR. Since this is a cross-team effort, I'm a bit unfamiliar with how you treat PRs, but we'd typically open a new issue and a new PR, as not to exceed the scope of the original PR.